libc/include/bits/fenv_riscv64.h

```diff
@@ -41,6 +41,7 @@ typedef unsigned int fenv_t;
 #define FE_OVERFLOW   0x04
 #define FE_DIVBYZERO  0x08
 #define FE_INVALID    0x10
+#define FE_DENORMAL   0x00 // compatibility
```

Comment:
DENORMAL is not defined in RISCV ISA F extension, please check Table 11.2: Accrued exception flag encoding. (I use 20191213, it is still the latest official version at <https://riscv.org/technical/specifications/> )